app/src/main/java/com/github/sdp/mediato/model/post/ReviewPost.java

```diff
@@ -5,60 +5,45 @@
 import androidx.annotation.Nullable;
 
 import com.github.sdp.mediato.model.Review;
+import com.github.sdp.mediato.model.media.Collection;
 
 import java.text.AttributedCharacterIterator;
 
 /**
  * Class representing a review post
  */
 public class ReviewPost extends Post{
-    private String title;
-    private int grade;
-    private String comment;
-    private int id;
+    private Review review;
+    private Collection collection;
 
-    private String mediaIconUrl;
-
-    public ReviewPost(String username, Review review){
+    public ReviewPost(String username, Review review, Collection collection) {
         super(REVIEW, username);
         //@TODO Add when username fixed for reviews

```

Comment:
is the todo to be addressed in this PR ?

Reply:
I removed it.